thirdeye/thirdeye-frontend/app/pods/components/rootcause-anomaly/component.js

```diff
@@ -49,6 +67,64 @@ export default Component.extend({
     }
   ),
 
+  /**
+   * Information about an anomaly's baselines, changes, and values to be displayed in the anomaly overview
+   * @type {Object}
+   * @example
+   * {
+   *   current: {
+   *    change: 1.1,
+   *    changeFormatted: +1.1,
+   *    value: 5000
+   *   }, {
+   *   baseline: {
+   *    change: 1.1,
+   *    changeFormatted: 1.1,
+   *    value: 1000
+   *   }
+   * }
+   */
+  anomalyInfo: computed(
+    'aggregates',
+    'anomalyUrns',
+    function () {
+      const { aggregates, anomalyUrns } = getProperties(this, 'aggregates', 'anomalyUrns');
+
+      const offsets = ['current', 'baseline', 'wo1w', 'wo2w', 'wo3w', 'wo4w'];
+      const metricUrns = filterPrefix(anomalyUrns, 'thirdeye:metric:');
+
+      if (!metricUrns) { return; }
+
+      let anomalyInfo = {};
+
+      [...offsets].forEach(offset => {
+        const change = aggregates[toOffsetUrn(metricUrns[0], 'current')] / aggregates[toOffsetUrn(metricUrns[0], offset)] - 1;
+        const roundedChange = (Math.round(change * 1000) / 10.0).toFixed(1);
+        const sign = change > 0 ? '+' : '';
+
+        let offsetFormatted = '';
+
+        switch(offset) {
+          case 'baseline':
+            offsetFormatted = 'Predicted Average';
+            break;
+          case 'current':
+            offsetFormatted = 'Current Average';
+            break;
+          default:
+            offsetFormatted = offset;
+        }
+
+        anomalyInfo[offsetFormatted] = {};
+        anomalyInfo[offsetFormatted].change = roundedChange;
+        anomalyInfo[offsetFormatted].value = aggregates[toOffsetUrn(metricUrns[0], offset)];
+        anomalyInfo[offsetFormatted].changeFormatted = `${sign}${roundedChange}`;
+      });
+
+      return anomalyInfo;
+    }
+  ),
+
   functionName: computed('anomaly', function () {
     return get(this, 'anomaly').attributes.function[0];
```

Comment:
I saw above that the `anomaly` property is a computed property that I guess returns an object, but it can also return a boolean as `false`. Will that be an error issue here?

Reply:
Good catch. I changed it to `return ;`. So in the anomaly computed property:
`if (!anomalyUrn || !entities || !entities[anomalyUrn]) return ;`